gems/pending/appliance_console/prompts.rb

```diff
@@ -85,6 +85,21 @@ def ask_for_ip_or_hostname_or_none(prompt, default = nil)
       ask_for_ip(prompt, default, validation, "a valid Hostname or IP Address.").gsub(/^'?NONE'?$/i, "")
     end
 
+    def ask_for_schedule_frequency(prompt = "Select schedule frequency: hourly, daily, weekly or monthly", default = "hourly")
```

Comment:
I'm not sure a default makes sense for these prompts. I would default the `default` parameter to `nil` and make the caller choose what the default should be for their particular use.